packages/apputils/src/kernelstatuses.tsx

```diff
@@ -58,8 +71,10 @@ function KernelStatusComponent(
   return (
     <TextItem
```

Comment:
This is a question: what do you all think about making `TextItem` and `GroupItem` default to adding an `onKeyDown` mirroring `onClick` when `onClick` is provided?

Reply:
Thinking: this would ensure downstream extensions are accessible too, without much overhead.

Reply:
Yeah i think this would be useful as this mighty be a functionality we would have create for other components down the line, so it would be good to add it as a default.

Reply:
Can this be done in this PR?

Reply:
currently working on how to implement this feature.

Reply:
Been looking at how to do this and cannot find a working mechanism. Any suggestions will be welcome. We cannot find a working solution without generating synthetic mouse clicks, which was rejected as a possible solution previously.